src/view/form/editing-form-view.js

```diff
@@ -99,6 +119,51 @@ export default class EditingFormView extends AbstractStatefulView {
     });
   };
 
+  #dateFromChangeHandler = ([userDate]) => {
+    this._setState({
+      point:{
+        ...this._state.point,
+        dateFrom:userDate
+      }});
+    this.#datePickerTo.set('minDate',this._state.point.dataFrom);
+  };
+
```

Comment:
десь проще использовать updateElement, тогда не придётся делать datePickerTo.set

```suggestion
  #dateFromChangeHandler = ([userDate]) => {
    this.updateElement({
      point: {
        ...this._state.point,
        dateFrom: userDate
      }
    });
  };

```

Reply:
аналогично с dateToChangeHandler